LocoSugarActivity.py

```diff
@@ -1,3 +1,4 @@
+from builtins import str
```

Comment:
Copyright must be before code.  Why is this import necessary?

Reply:
I have done porting by Futurize not by 2to3. Futurize automatically import some statements. These are un-necessary I guess. I will remove them.

Reply:
Thanks.  I ask that porting is done using 2to3 unless there is a good reason for using Futurize.  At this stage, I don't know if you have a reason.